bob.js

```diff
@@ -949,6 +955,14 @@ leaderboardCmd = (message) => {
     }
 }
 
+// !nr/!newrunner
+const runnerPrefixes = ["KaDi", "p-p", "Rbd",       "Liam",  "Wiigo", "Slen", "Krosso", "Gen",  "Loud",   "Steel",    "MJG", "teddy", "Darkened", "bross", "Fire",   "S0ul",  "super", "Orca",  "Ads",  "Sean",    "King", "Panda",   "The",      "Mzze", "Black",    "Creator", "a50_Caliber", "ItsDa", "Retro",     "tip",     "laki", "Fr",  "A2on",  "Obed", "AB", "Shrimpii", "xsHI", "Lombax",  "Plop", "Azure",    "MyPair",  "Sky",     "Inclusion", "MajorLeague", "TheReal", "stoic", "Gel", "Mega_Mario", "Pro", "Dbp",    "dyn",  "Glitch",  "fri", "Ricky", "pringles", "TylerThe", "Ril", "artic",    "Real",     "Zachi"];
```

Comment:
If we're gonna start adding fun commands, maybe we should separate it out into its own file so it's separate from all the race stuff.